dizoo/classic_control/cartpole/config/cartpole_dqn_config.py

```diff
@@ -24,6 +24,7 @@
             update_per_collect=5,
             batch_size=64,
             learning_rate=0.001,
+            learner=dict(hook=dict(save_ckpt_after_iter=100)),
```

Comment:
why add this

Reply:
because when we do unit test at drex, we need to modify the` learner.hook.save_ckpt_after_iter`, if we do not have this, the unit test will be failed, so I add this.